sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -48,4 +51,47 @@ export class ConfirmationOfEnrollmentController {
       };
     }) as COESummaryDTO[];
   }
+
+  /**
+   * Creates a new Student Application overriding the current one
+   * in order to rollback the process and start the assessment
+   * all over again.
+   * @param locationId location id executing the COE rollback.
+   * @param applicationId application to be rolled back.
+   * @returns the id of the newly created Student Application.
+   */
+  @HasLocationAccess("locationId")
+  @Post(":locationId/confirmation-of-enrollment/:applicationId/rollback")
```

Comment:
how about `@Post(":locationId/confirmation-of-enrollment/application/:applicationId/rollback")` @andrewsignori-aot  @guru-aot  @dheepak-aot thoughts?

Reply:
That sounds more descriptive and easy to understand to me. Happy to do the change 😉

Reply:
I did the change, @guru-aot and @dheepak-aot, please let me know if you have some different ideas or suggestions.

Reply:
@andrewsignori-aot I'm good with Ann's suggestion.